### ISSUE TITLE
can not train the caffenet with your solver

### ISSUE THREAD

**Issue by xinxiaomeng** [D-X-Y/caffe-reid#7]
when I train the caffenet with your solver, the loss is always 87.3365. Then I make the base_lr=0.001, the loss is reduce. However, I can not get the same precision with you. I get MAP=0.24, r1=0.43 when the Iteration is 78000, the loss=1.80 loss_dif=0.31

**Comment by liuajian** [comment]
@xinxiaomeng my loss is always 87.3365 too. how to set base_lr,  the paper is 0.001 original

**Comment by D-X-Y** [comment]
Maybe you can try https://github.com/D-X-Y/caffe-reid/tree/71fb8f574ef942692858d1bcdcaee586706dc137 , I change the prototype generation codes many times and there might be some issues, so please use previous versions.

**Comment by tofulife** [comment]
I have used the previous version just now ,but the loss was 87.3365 either.

**Comment by liuajian** [comment]
@tofulife I same with you

**Comment by D-X-Y** [comment]
@liuajian @tofulife @xinxiaomeng Sorry for the late reply. Recently, I focus on preparing the CVPR 2018, and am too busy to check the problems. I will come back after the CVPR due and fix the problem then.

**Comment by sillytheif** [comment]
@xinxiaomeng I couldn't fint the pre-trained model and data,shall I download it myself or was it offered by the code?

**Comment by ssFade** [comment]
my loss is always 87.3365 too...

**Comment by D-X-Y** [comment]
@xinxiaomeng @liuajian @sillytheif @ssFade @tofulife  Hi, sorry for the late reply. I change the loss weight of the identity loss and the classification loss, which is currently not consistent with the original paper. However, the new loss weights (0.5 for identity and 1 for classification) works well on the caffe framework. It should work fine and get the correct results now.
